### PR TITLE
Fixed some issues I had when using SassyJSON

### PR DESCRIPTION
**Pull request by esr360**
I ran into a couple of issues when using SassyJSON, the commits in this request fixed these issues.

### CODE REVIEW

**Comment by esr360** [comment]
Regarding the 'deprecation warning: Unescaped multiline' commit, I have to admit I'm unsure why a new line was initially used here, but an empty quote here has yielded me perfect to JSON results.

File: stylesheets/encode/helpers/_quote.scss [hunk 1]
```diff
@@ -2,6 +2,11 @@
 /// @access private
 /// @param {*} $value - value to be quoted
 /// @return {String} - quoted value
+
+// [1] if $value is 'null', Sass returns 'Invalid null operation: "'"' plus null"',
```
Comment (HugoGiraudel) [inline stylesheets/encode/helpers/_quote.scss]: This comment might be causing issues with SassDoc. I get why you convert it to string before, you can safely remove it. ;)

File: stylesheets/decode/types/_string.scss [hunk 2]
```diff
@@ -21,8 +21,7 @@
     // If string is not empty
     $string: str-slice($temp, 1, $end - 1);
 
-    $cr: "
-    ";
+    $cr: "";
```
Comment (HugoGiraudel) [inline stylesheets/decode/types/_string.scss]: This is wrong unfortunately, and there is no good fix for this.

Comment (esr360) [inline stylesheets/decode/types/_string.scss]: Fair enough, that's a shame. For what it's worth, it seems to be working perfectly for me :) But frankly I have no idea what this code is supposed to be doing.

File: stylesheets/encode/helpers/_quote.scss (hunk 1 above)
Comment (esr360) [inline stylesheets/encode/helpers/_quote.scss]: Cool, done!

**Pull request merged**